website/docs/Publishing.md

```diff
@@ -46,4 +48,30 @@ Even if you are using Github Pages, and:
 
 ## Github Pages Custom Domain CNAME file
 
-Github Pages requires you to create a CNAME file in your website to host on a custom domain. You should create this file at `yourProjectRoot/.moonwave/static/CNAME` (create the folders if they don't exist) and put your domain in the file. 
\ No newline at end of file
+Github Pages requires you to create a CNAME file in your website to host on a custom domain. You should create this file at `yourProjectRoot/.moonwave/static/CNAME` (create the folders if they don't exist) and put your domain in the file. 
+
+## Using GitHub Actions
+
+You can create a GitHub Actions workflow to automatically publish your doc webiste to GitHub Pages. To begin, you need to create a new workflow in the `.github/workflows` directory of your repository that will build and publish your website.
+
+Here is an example of such workflow where the job will run on each push to the `master` branch:
+
+```yaml
+# .github/workflows/publish.yml
+on:
+  push:
+    branches: ["master"]
+jobs:
+  status:
+    runs-on: ubuntu-latest
+    name: Publish doc to GitHub Pages
+    steps:
```

Comment:
Make sure you checkout the repository, setup node, and install Moonwave too, otherwise this example wouldn't work.

Reply:
Dang, thanks for flagging!

Reply:
Don't forget to also setup node and install Moonwave from npm.

```yml
- uses: actions/checkout@v3
- uses: actions/setup-node@v4
  with:
    node-version: "18"
- run: npm i -g moonwave@latest
```

Reply:
Good point! I was using `npx moonwave ...` in makefile. Does it makes a difference installing it globally?